.github/workflows/test-suite.yml

```
@@ -73,7 +80,7 @@ jobs:
     with:
       terraform_version: ${{ matrix.terraform_version }}
       provider_version: ${{ matrix.provider_version }}
-      atlas_cloud_env: ${{ inputs.atlas_cloud_env || github.event.schedule == '0 0 2-31 * 0' && 'qa' || 'dev' }}
+      atlas_cloud_env: ${{ inputs.atlas_cloud_env || needs.day_of_week.outputs.day == 'Sun' && 'qa' || 'dev' }}
```

```suggestion
      atlas_cloud_env: ${{ inputs.atlas_cloud_env || needs.day_of_week.outputs.day == 'Sun' && 'qa' }}
```

acceptance-tests defaults to dev, no need to pass it explicetely

---

Applied in 9a16a1a16fbb846225446ca94c1a33e314c75fd2